tools/gulptasks/dashboards/api-docs.json

```diff
@@ -78,6 +78,8 @@
         "../../../ts/DataGrid/TableHead.ts",
         "../../../ts/DataGrid/Globals.ts",
         "../../../ts/DataGrid/Utils.ts",
+        "../../../ts/DataGrid/Querying/QueryingController.ts",
+        "../../../ts/DataGrid/Querying/SortingController.ts",
```

Comment:
I'm not sure if these two should be public 🤔 
We have `setOrder` method that covers most of the cases...